client/RoomClient.js

```diff
@@ -1617,16 +1674,46 @@ export default class RoomClient extends EventTarget
         	key,
         	value,
         });
-    }
+	}
+	
+	getFile(key) {
```

Comment:
Shouldn't this be a promise that resolves when the file is downloaded? Also... could this be a separate rest call to not clog the websocket?

Reply:
outdated